src/test/java/BrokenImageBMPTest.java

```diff
@@ -0,0 +1,43 @@
+import net.lightbody.bmp.core.har.Har;
+import net.lightbody.bmp.core.har.HarEntry;
+import net.lightbody.bmp.proxy.ProxyServer;
+import org.testng.Assert;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+import core.BmpTestBase;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import static com.codeborne.selenide.Selenide.$$;
+import static com.codeborne.selenide.Selenide.open;
+import static helpers.Helpers.getUrls;
+import static helpers.Helpers.mapToString;
+
+public class BrokenImageBMPTest extends BmpTestBase {
+    private ProxyServer bmp;
+
+    @BeforeMethod
+    public void setup() {
+        bmp = getServer();
+        open("broken_images");
+    }
+
+    @Test
+    public void brokenImageTest() {
+        List<String> links = getUrls($$(".example>img"), "src");
```

Comment:
That is totally not your case. Have a look at net panel in firebug when the the pages is loaded - you will see the responses that failed -> so check them in hard